src/tests/test_utils/functional_test_utils/layer_tests_summary/analyze_confomance_fails.py

```diff
@@ -79,6 +79,15 @@ def process_remote_log(self, remote_log_path, job_list):
                             zipObj.extract(name, dest_path)
                 logger.info(f'DONE')
 
+    def get_real_device(self, device):
```

Comment:
Why?

Reply:
in text logs of DGPU tests is called like [entity]/[case].[test]/targetDevice=GPU  , script should find 'GPU', replace it to HWDevice and put status, but it can not and in the end we have two rows in table: 
[entity]/[case].[test]/targetDevice=HWDevice - for all devices with current status, but not for DGPU
[entity]/[case].[test]/targetDevice=GPU - for all devices status is 'not run' , for DGPU pass/fail